lib/core.ps1

```diff
@@ -588,10 +588,19 @@ function Invoke-ExternalCommand {
     return $true
 }
 
-function env($name,$global,$val='__get') {
-    $target = 'User'; if($global) {$target = 'Machine'}
-    if($val -eq '__get') { [environment]::getEnvironmentVariable($name,$target) }
-    else { [environment]::setEnvironmentVariable($name,$val,$target) }
+function env($name, $global, $val='__get') {
+    $target = 'User'
+    $EnvironmentRegisterKey = 'HKCU:\Environment'
+    if ($global) {
+        $target = 'Machine'
```

Comment:
Both `$target` variables are no longer used

Reply:
I forgot to modify it, thanks for pointing it out.